source/manual/transition-a-site.html.md

```diff
@@ -136,6 +136,9 @@ Domains at the root of their zone can't use `CNAME` records, so must use an `A`
 record and point at one of the [Fastly GOV.UK IP
 addresses](https://github.com/alphagov/transition/blob/016c3d30e190c41eaa912ed554384a49f3418a91/app/models/host.rb#L22).
 
+If the site is one that was [administered by GDS](https://github.com/alphagov/gds-dns-config/tree/master/zones)
+(e.g. theorytest.direct.gov.uk), you will also need to [update and re-deploy the DNS config](/manual/dns.html#making-changes-to-publishing-service-gov-uk).
```

Comment:
I'd remove `also` from here, as updating the DNS is the only thing you'll need to do to point the domain at us/bouncer.